workos/utils/request.py

```diff
@@ -9,7 +11,10 @@
 )
 
 BASE_HEADERS = {
-    "User-Agent": "WorkOS Python/{}".format(workos.__version__),
+    "User-Agent": "Python/{} WorkOS Python SDK/{}".format(
```

Comment:
Could we keep the "WorkOS" first? Again, to stay consistent with the Ruby SDK:

```suggestion
    "User-Agent": "WorkOS Python/{} Python SDK/{}".format(
```